src/plugins/intel_cpu/src/nodes/executors/acl/acl_pooling.cpp

```diff
@@ -55,8 +55,9 @@ bool AclPoolingExecutor::isSupported(const TensorInfo& srcTensorInfo,
         DEBUG_LOG("NCHW + CEIL gives an accuracy problem in ACL AvgPool. ACL executor will not be created.");
         return false;
     }
+    // TORCH_CEIL is mapped to ACL CEIL type
     DimensionRoundingType round =
```

Comment:
Lets do explicit mapping: FLOOR -> FLOOR, CEIL/TORCH_CEIL -> CEIL, others - not supported

Reply:
Fixed